react/features/authentication/index.js

```diff
@@ -1,3 +1,4 @@
 export * from './actions';
 export * from './actionTypes';
-export * from './components';
+export * from './components/native';
+export * from './components/web';
```

Comment:
This won't work. Create an index file for web and native if need be, check other features. I'd also be fine dropping them from the index, since it can be a source of problems.

Reply:
This was deleted.